pom.xml

```diff
@@ -186,13 +196,6 @@
         <type>pom</type>
       </dependency>
       <dependency>
-        <groupId>org.infinispan</groupId>
```

Comment:
The Spring BOM was declared before Infinispan BOM and that caused some Infinispan artifacts to be of that version, when I moved Infinispan BOM before Spring BOM the opposite occured (some of the Spring dependency versions were taken from Infinispan BOM), could not find a more elegant and correct way of making it work 😞